wgpu-core/src/device/mod.rs

```diff
@@ -852,7 +852,16 @@ impl<A: HalApi> Device<A> {
             ));
         }
 
-        // TODO: validate missing TextureDescriptor::view_formats.
+        if let Some(ref view_formats) = desc.view_formats {
+            for format in view_formats {
+                if desc.format == *format {
```

Comment:
Does the spec require that the format list is unique?

Reply:
If I understand correctly, the spec does not require uniqueness of format list values.